EnglishFilter/sections/specific.txt

```diff
@@ -164,6 +164,8 @@ kissanime.ac###adlink2
 ||vcdn.rivertraffic.com/*.mp4$domain=viptube.com
 masseurporn.com##.video_cums
 masseurporn.com##.video_cums1
+faapy.com##a[href="https://faapy.com/cs/mylf/"]
+faapy.com##.container > .textlink
```

Comment:
use more specific selector in this case. Something like this
`.footer > div.container > div.textlink`